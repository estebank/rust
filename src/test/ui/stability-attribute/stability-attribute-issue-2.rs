#![feature(staged_api)]
#![stable(feature = "test", since = "0")]

#[stable(feature = "test", since = "0")]
pub struct A<T>(pub T); //~ ERROR field has missing stability attribute

fn main() {
    // Make sure the field is used to fill the stability cache
    A(0).0;
}
