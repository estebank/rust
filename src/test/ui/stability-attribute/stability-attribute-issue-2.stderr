error: field has missing stability attribute
  --> $DIR/stability-attribute-issue-2.rs:5:17
   |
LL | pub struct A<T>(pub T);
   |                 ^^^^^

error: aborting due to previous error

